client/src/Components/DataFetch.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import axios from 'axios';
+
+class DataFetch extends React.Component {
+
+  constructor() {
+    super();
+    this.state = {
```

Comment:
shorterhand would be 

state= {
players:[]
}

you do not need to have constructor or super